presto-parquet/src/main/java/com/facebook/presto/parquet/reader/PrimitiveColumnReader.java

```diff
@@ -266,17 +268,19 @@ private void updateValueCounts(int valuesRead)
 
     private ValuesReader readPageV1(DataPageV1 page)
     {
-        ValuesReader rlReader = page.getRepetitionLevelEncoding().getValuesReader(columnDescriptor, REPETITION_LEVEL);
-        ValuesReader dlReader = page.getDefinitionLevelEncoding().getValuesReader(columnDescriptor, DEFINITION_LEVEL);
-        repetitionReader = new LevelValuesReader(rlReader);
-        definitionReader = new LevelValuesReader(dlReader);
+        ValuesReader repetitionLevelReader = page.getRepetitionLevelEncoding().getValuesReader(columnDescriptor, REPETITION_LEVEL);
+        ValuesReader definitionLevelReader = page.getDefinitionLevelEncoding().getValuesReader(columnDescriptor, DEFINITION_LEVEL);
+        repetitionReader = new LevelValuesReader(repetitionLevelReader);
+        definitionReader = new LevelValuesReader(definitionLevelReader);
         try {
             byte[] bytes = page.getSlice().getBytes();
-            rlReader.initFromPage(page.getValueCount(), bytes, 0);
-            int offset = rlReader.getNextOffset();
-            dlReader.initFromPage(page.getValueCount(), bytes, offset);
-            offset = dlReader.getNextOffset();
-            return initDataReader(page.getValueEncoding(), bytes, offset, page.getValueCount());
+            ByteBufferInputStream bufferInputStream = ByteBufferInputStream.wrap(ByteBuffer.wrap(bytes));
```

Comment:
There are var args calls here, I would use lists for those:

```
ByteBufferInputStream.wrap(ImmutableList.of(ByteBuffer.wrap(bytes)));
```

Reply:
Done.